app/scripts/services/tile-proxy.js

```diff
@@ -560,7 +561,9 @@ export const tileDataToPixData = (
     pseudocount,
     colorScale,
     ignoreUpperRight,
-    ignoreLowerLeft
+    ignoreLowerLeft,
+    tile.tileData.shape,
+    tile.selectedRows
```

Comment:
I'm not sure that `selectedRows` should be a property of the tile. It's an option like `ignoreUpperRight` and `ignoreLowerLeft` that applies to all tiles equally, right?

Reply:
Yes that is correct